app/_landing_pages/gateway/deployment-topologies.yaml

```diff
@@ -8,12 +8,224 @@ rows:
+            description: |
+              Learn about Dedicated Cloud Gateways hosting for Data Planes.
+              
+            icon: /assets/icons/dedicated-cloud-gateway.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#dedicated-cloud-gateways"
+      - blocks:
+        - type: card
+          config:
+            title: Self-hosted Data Plane nodes
+            description: |
+              Learn about self-hosting Data Plane nodes.
+              
+            icon: /assets/icons/self-hosted.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#self-hosted"
+    
+  - header:
+      type: h2
+      text: "Traditional mode"
+    columns:
+      - blocks:
+          - type: mermaid
+            config:
+              diagram: |
+                flowchart TD
+
+                A[(Database)]
+                B(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Kong Gateway instance)
+                C(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Kong Gateway instance)
+                D(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Kong Gateway instance)
+
+                A <---> B & C & D
+
+                style B stroke:none,fill:#0E44A2,color:#fff
+                style C stroke:none,fill:#0E44A2,color:#fff
+                style D stroke:none,fill:#0E44A2,color:#fff
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    In traditional mode, {{site.base_gateway}} requires a database to store configured entities such as Routes, Services, and plugins.
+                    In a traditional deployment, all {{site.base_gateway}} nodes connect to the database. Each node manages its own configuration. 
+
+                    It is the only deployment topology that supports plugins that require a database, like rate-limiting with the cluster strategy, or OAuth2. 
+                    However, there are some downsides too.
+                    When running in traditional mode, every {{ site.base_gateway }} node runs as both a Control Plane (CP) and Data Plane (DP). This means that if **any** of your nodes are compromised, the entire running gateway configuration is compromised. 
+
+                    In addition, if you're running {{site.ee_product_name}} with [Kong Manager](/gateway/kong-manager/), request throughput may be reduced on nodes running Kong Manager due to expensive calculations being run to render analytics data and graphs.
+
+                    **Works on:** {{site.base_gateway}}
+
+          - type: button
+            config:
+              align: left
+              text: Learn more about traditional mode
+              url: /gateway/traditional-mode/
+  - header:
+      type: h3
+      text: "Supported Data Plane hosting options"
+    columns: 
+      - blocks:
+        - type: card
```

Comment:
Non-blocking: We should have a way to set a max width on these cards 
![CleanShot 2025-03-20 at 11 12 04](https://github.com/user-attachments/assets/fa39f74b-27b8-45ac-acd9-43abef74b3ea)

Reply:
I added this to a list I shared of platform improvements: https://kongstrong.slack.com/archives/C06TLRHG8KX/p1742484862727729